src/containers/Notes/Notes.tsx

```diff
@@ -95,7 +104,21 @@ class Notes extends React.Component<{}, NotesState> {
         this.setState({ ...this.state, isDarkMode: true });
     }
 
+    private getSearchInputValue(input: string): void {
+        const filteredArray = this.filterArrayBySearchValue(input);
+        this.setState({ ...this.state, searchNotes: filteredArray, searchValue: input });
+    }
+
+    public filterArrayBySearchValue = (input: string): Note[] => {
+        const arrayFilteredBySearchValue = this.state.notes.filter((note) => {
+            return note.title.toLowerCase().includes(input.toLowerCase()) ? true : false;
```

Comment:
you don't need a ternary operator here, as includes() method returns boolean value, so it could be just `
const arrayFilteredBySearchValue = this.state.notes.filter((note) => note.title.toLowerCase().includes(input.toLowerCase())
`